.circleci/config.yml

```diff
@@ -996,7 +1013,7 @@ jobs:
           path: /home/circleci/project/
       - run:
           name: Building wheel for <<parameters.project_name>>
-          command: python setup.py bdist_wheel
+          command: PYENV_VERSION=$(pyenv global) python setup.py bdist_wheel
```

Comment:
For the record, this change is to ensure we don't try to use a locally-set python version for building the wheel, because the local python version might not be installed (as is the case now for flowetl).

Ideally we'd just set '3.8' as the flowetl local python version, but the pyenv installed on the machine used for the flowetl integration tests doesn't support partial version specification in .python-version (and also doesn't support installing python 3.8.16, which is the version installed in the image used here).

I'm not sure whether this is the best solution.